src/Template/Element/Form/other_properties.twig

```diff
@@ -1,26 +1,110 @@
 <script type="text/javascript">
-$(document).ready(function(){
-    $('textarea.json').change(function(event){
-        if (event && event.target && event.target.value) {
-            try {
-                let jsonString = JSON.stringify(JSON.parse(event.target.value), null, "    ");
-                $(event.target).val(jsonString);
-            } catch(e) {
-                console.error(e);
-            }
+
+    let isValidKey = (e) => {
+        var scope = this;
+        var k = e.which;
+
+        // Verify that the key entered is not a special key
+        if (k == 20 /* Caps lock */
+            || k == 16 /* Shift */
+            || k == 9 /* Tab */
+            || k == 27 /* Escape Key */
+            || k == 17 /* Control Key */
+            || k == 91 /* Windows Command Key */
+            || k == 19 /* Pause Break */
+            || k == 18 /* Alt Key */
+            || k == 93 /* Right Click Point Key */
+            || ( k >= 35 && k <= 40 ) /* Home, End, Arrow Keys */
+            || k == 45 /* Insert Key */
+            || ( k >= 33 && k <= 34 ) /*Page Down, Page Up */
+            || (k >= 112 && k <= 123) /* F1 - F12 */
+            || (k >= 144 && k <= 145 )) { /* Num Lock, Scroll Lock */
+            return false;
         }
-    });
-    $('textarea.json').each(function(){
-        if (this.value) {
-            try {
-                let jsonString = JSON.stringify(JSON.parse(this.value), null, "    ");
-                $(this).val(jsonString);
-            } catch(e) {
-                console.error(e);
+        return true;
+    }
+
+    let cleanupForm = () => {
+        // reset messages/styles for password validation
+        let form = document.getElementById('form--main');
+        let confirm_password = form.elements.confirm_password;
+        confirm_password.classList.remove('error', 'empty', 'different');
+        confirm_password.closest('div').removeAttribute('message-tooltip');
+    }
+
+    let validateForm = (event, callback) => {
+        event.preventDefault();
+        event.stopPropagation();
+
+        let form = document.getElementById('form--main');
+        let passwordGroup = form.elements.password;
+        let password = passwordGroup[0];
```

Comment:
This causes a JS error on `Save` in other modules like `Documents` or any other where no `password` field is present.

```
14085:380 Uncaught TypeError: Cannot read property '0' of undefined
    at HTMLInputElement.validateForm (14085:380)
validateForm @ 14085:380
```